torch/_torch_docs.py

```diff
@@ -3152,7 +3152,16 @@ def merge_dicts(*dicts):
            r"""
 flatten(input, start_dim=0, end_dim=-1) -> Tensor
 
-Flattens a contiguous range of dims in a tensor.
+Returns tensor with the same data and number of elements as :attr:`input`,
+but with dimensions in [:attr:`start_dim`, :attr:`end_dim`] collapsed into a
+single dimension. When possible, the returned tensor will be a view of :attr:`input`.
+Otherwise, it will be a copy.
+
+See :meth:`torch.Tensor.view` on when it is possible to return a view.
+
+.. note::
+    Flattening a scalar (zero-dimensional tensor) is defined to return a size one
```

Comment:
"Flattening a scalar (zero-dimensional tensor) is defined to return a size one" -> "Flattening a zero-dimensional tensor will return a one-dimensional tensor"